src/lib_c/x86_64-windows-msvc/c/winapi.cr

```diff
@@ -215,17 +215,17 @@ lib LibC
 
   # source https://msdn.microsoft.com/en-us/library/windows/desktop/ms725481(v=vs.85).aspx
   struct TIME_ZONE_INFORMATION
-    _Bias : Long
-    _StandardName : StaticArray(WChar, 32)
-    _StandardDate : SystemTime
-    _StandardBias : Long
-    _DaylightName : StaticArray(WChar, 32)
-    _DaylightDate : SystemTime
-    _DaylightBias : Long
+    bias : Long
+    standard_name : StaticArray(WChar, 32)
+    standard_date : SystemTime
+    standard_bias : Long
+    daylight_name : StaticArray(WChar, 32)
+    daylight_date : SystemTime
+    daylight_bias : Long
   end
 
-  fun _GetTimeZoneInformation = GetTimeZoneInformation(tz_info : TIME_ZONE_INFORMATION*) : DWord
-  fun _GetSystemTimeAsFileTime = GetSystemTimeAsFileTime(time : FILETIME*)
+  fun get_time_zone_information = GetTimeZoneInformation(tz_info : TIME_ZONE_INFORMATION*) : DWord
+  fun get_system_time_as_file_time = GetSystemTimeAsFileTime(time : FILETIME*)
```

Comment:
What's the difference with winbase.cr  did I miss something ? Imo these modifications in this file are wrong

Reply:
Indeed, that file was commited on error.